sources/packages/api/src-sql/Reports/Create-disbursement-report.sql

```diff
@@ -0,0 +1,44 @@
+--Insert config for disbursement report.
+INSERT INTO
+    sims.report_configs (report_name, report_sql)
+VALUES
+    (
+        'Disbursement_Report',
+        'SELECT "Date of Disbursement",
+            "SIN",
+            "Application Number",
+            "Certificate Number",
+            "Funding Code",
+            "Disbursement amount"
+          FROM
+            (SELECT TO_CHAR(DR.DISBURSE_DATE, ''YYYY-MM-DD'') AS "Date of Disbursement",
+                DR.STUDENT_SIN AS "SIN",
+                SA.APPLICATION_ID AS "Application Number",
+                DS.DOCUMENT_NUMBER AS "Certificate Number",
+                DRV.GRANT_TYPE AS "Funding Code",
+                DRV.GRANT_AMOUNT AS "Disbursement amount",
+                EPO.OFFERING_INTENSITY AS "Offering intensity"
+              FROM SIMS.DISBURSEMENT_RECEIPTS DR
+              INNER JOIN SIMS.DISBURSEMENT_RECEIPT_VALUES DRV ON DRV.DISBURSEMENT_RECEIPT_ID = DR.ID
+              INNER JOIN SIMS.DISBURSEMENT_SCHEDULES DS ON DS.ID = DR.DISBURSEMENT_SCHEDULE_ID
+              INNER JOIN SIMS.STUDENT_ASSESSMENTS SA ON SA.ID = DS.STUDENT_ASSESSMENT_ID
+              INNER JOIN SIMS.EDUCATION_PROGRAMS_OFFERINGS EPO ON EPO.ID = SA.OFFERING_ID
+              UNION ALL SELECT TO_CHAR(DR.DISBURSE_DATE, ''YYYY-MM-DD'') AS "Date of Disbursement",
+                DR.STUDENT_SIN AS "SIN",
+                SA.APPLICATION_ID AS "Application Number",
+                DS.DOCUMENT_NUMBER AS "Certificate Number",
+                CASE
+                        WHEN DR.FUNDING_TYPE = ''BC'' THEN ''BCSL''
+                        WHEN DR.FUNDING_TYPE = ''FE'' THEN ''CSL''
+                END AS "Funding Code",
+                DR.TOTAL_DISBURSED_AMOUNT AS "Disbursement amount",
+                EPO.OFFERING_INTENSITY AS "Offering intensity"
+              FROM SIMS.DISBURSEMENT_RECEIPTS DR
+              INNER JOIN SIMS.DISBURSEMENT_SCHEDULES DS ON DS.ID = DR.DISBURSEMENT_SCHEDULE_ID
+              INNER JOIN SIMS.STUDENT_ASSESSMENTS SA ON SA.ID = DS.STUDENT_ASSESSMENT_ID
+              INNER JOIN SIMS.EDUCATION_PROGRAMS_OFFERINGS EPO ON EPO.ID = SA.OFFERING_ID) AS DATA
+          WHERE "Offering intensity" = ANY(:offeringIntensity)
```

Comment:
How DB is resolving this with the `where` clause outside the queries?
Is it going to resolve both queries and then apply the result to the final combination of both?
If this is the case, the execution plan would not be better for executing the where on each one of the queries?
Was it checked?

Reply:
Yes its resolving, both the queries and apply the where clause to the result. I checked, it is the same as executing them as separate queries.

Reply:
As discussed changed